pandas/core/generic.py

```diff
@@ -3756,6 +3756,18 @@ class   animal   locomotion
         """
         axis = self._get_axis_number(axis)
         labels = self._get_axis(axis)
+
+        if isinstance(key, list):
+            len_levels = 1 if level is None or is_scalar(level) else len(level)
+            if len(key) != len_levels:
```

Comment:
does this break any existing tests?

Reply:
One ``test_xs_partial``

Could deprecate this too, if you prefer?

Reply:
yeah let's just deprecate entirely (prob easier to remember too)

Reply:
Done